release/NativeEditPlugin/scripts/NativeEditBox.cs

```diff
@@ -238,25 +237,14 @@ private void PrepareNativeEdit()
 		mConfig.multiline = (objUnityInput.lineType == InputField.LineType.SingleLine) ? false : true;
 	}
 
-	private void onTextChange(string newText)
+	private void updateUnityInputText(string newText)
 	{		
 		// Avoid firing a delayed onValueChanged event if the text was changed from Unity with the text property in this
 		// class.
 		if (newText == this.objUnityInput.text)
 			return;
```

Comment:
This can be removed too as there is only an assignment in this function

Reply:
Oh yes, we're not manually triggering it anymore. Will also remove this check 👍